lib/db/schema/patch-023-024.sql

```diff
@@ -0,0 +1,175 @@
+-- Add table for storing unverified sessionTokens.
+CREATE TABLE `unverifiedTokens` (
+  tokenId BINARY(32) NOT NULL PRIMARY KEY,
+  tokenVerificationId BINARY(16) NOT NULL,
+  uid BINARY(16) NOT NULL
+) ENGINE=InnoDB;
```

Comment:
(or perhaps better, on `(uid, tokenVerificationId)` so that we can easily list all unverified tokens belonging to a particular user)